### ISSUE TITLE
Random formula generator ignores `--dim` option when `--seed` is setted

### ISSUE THREAD

**Issue by caramellaio** [black-sat/black#70]
Hi,

I noticed that `--seed` option creates deterministic formula but ignores the option `--dim`.

```
> random_formulas_generator --seed 1024 --dim 1024
F((((Z (p3) W (!!((p4) | H (p3)) | (p4))) M (F wX wX (p2) S ((p2) S (p2)))) U O((G X True -> (p4)) M ((p5) R Z Y Y((F True M Y H((p5) R Z (p3))) S X !(p4))))) T ((F G((p5) | (p1)) -> ((p5) U H X((p2) <-> wX X !((p3) T (p6))))) M (p1)))

> random_formulas_generator --seed 1024 --dim 102
F((((Z (p3) W (!!((p4) | H (p3)) | (p4))) M (F wX wX (p2) S ((p2) S (p2)))) U O((G X True -> (p4)) M ((p5) R Z Y Y((F True M Y H((p5) R Z (p3))) S X !(p4))))) T ((F G((p5) | (p1)) -> ((p5) U H X((p2) <-> wX X !((p3) T (p6))))) M (p1)))

```

**Comment by GabVenturato** [comment]
Ah right, my bad. I'll fix it asap.

**Comment by nicola-gigante** [comment]
Thanks for noticing! Fixed by #71 just merged.
